fcl/reco/Definitions/stage1_icarus_defs.fcl

```diff
@@ -65,9 +68,11 @@ icarus_stage1_producers:
 
  ## crt producer
   crttrack:          @local::standard_crttrackproducer
-  CRTT0Matching:     @local::standard_crtt0producer
-  CRTT0MatchingW:    @local::standard_crtt0producerW
-  CRTT0MatchingE:    @local::standard_crtt0producerE
+  ### FPoppi: the following are deprecated (and the one above not mantained)
```

Comment:
My suggestion is if the code no longer exists or is supported that it be removed from  here to avoid future confusion.

Reply:
Ok Tracy, I am going to remove it!